js-miniapp-sdk/src/miniapp.ts

```diff
@@ -308,4 +308,8 @@ export class MiniApp implements MiniAppFeatures, Ad, Platform {
   getPermissionStatus(name: PermissionName): Promise<string> {
     return getBridge().utilityManager.getPermissionStatus(name);
   }
+
+  launchAppSettings(): Promise<boolean> {
+    return getBridge().utilityManager.launchAppSettings();
+  }
```

Comment:
Do you think we can move this inside MiniAppUtils would be good ? Or new manager in SDK?
Because right now, if the MiniApps call MiniApp.launchSettings() would be kinda misleading you think ? Like its launching miniapp settings, but actually it is launching device settings.